pennylane/init.py

```diff
@@ -20,6 +20,66 @@
 from pennylane import numpy as np
 
 
+def particle_conserving_u1_uniform(n_layers, n_wires, low=0, high=2 * pi, seed=None):
+    r"""Creates a parameter array for :func:`~.ParticleConservingU1`, drawn from a uniform
+    distribution.
+
+    Each parameter is drawn uniformly at random from between ``low`` and ``high``.
+    The parameters define the trainable angles entering the particle-conserving
+    exchange gates :math:`U_{1,\mathrm{ex}}(\phi, \theta)` implemented by the
+    :func:`~.u1_ex_gate()`.
+
+    Args:
+        n_layers (int): number of layers
+        n_wires (int): number of qubits
+        low (float): minimum value of uniform distribution
+        high (float): maximum value of uniform distribution
+        seed (int): seed used in sampling the parameters, makes function call deterministic
```

Comment:
I think @mariaschuld commented also on this regarding the refactoring. As I see it, by passing a seed the user makes sure the initial set of parameters is the same. It may be useful to test the optimizers, right?.